pyproject.toml

```diff
@@ -45,7 +45,8 @@ classifiers = [
 dependencies = [
     "torch>=2.3,<3",
     "numpy>=1.17",
-    "packaging>=20.9"
+    "packaging>=20.9",
+    "kernels>=0.11.1"
```

Comment:
I think we could make this optional, e.g. pull in if user has `kernels` and fallback otherwise.